Makefile

```diff
@@ -349,6 +349,14 @@ test: ##@test Run tests once in NodeJS
 	yarn shadow-cljs compile test && \
 	node --require ./test-resources/override.js target/test/test.js
 
+android-test: jsbundle
+android-test: export TARGET := android
+android-test:
+	yarn install
+	yarn shadow-cljs compile mocks && \
```

Comment:
Although I'm not 100% sure if this is needed

Reply:
I mean compiling mocks and a target of jsbundle

Reply:
not needed, I meant to remove it, jsbundle is needed (or rather, it doesn't work otherwise, but it might be possible not to run it, not sure)

Reply:
> it might be possible not to run it

interesting! worth exploring. Thanks for the explanation.